src/services/httpApi/dtos.ts

```diff
@@ -101,6 +101,21 @@ export class VerifyChannelResponse {
 
   // ID of the verified user
   @IsString() @ApiProperty({ required: true }) userId: string
+
+  // Youtube Channel/User handle
+  @IsString() @ApiProperty() channelHandle: string
+
+  // Youtube Channel title
+  @IsString() @ApiProperty({ required: true }) channelTitle: string
+
+  // Youtube Channel description
+  @IsString() @ApiProperty({ required: true }) channelDescription: string
```

Comment:
should this be not required? When it's not provided(on yt side) the backend returns a blank string. On the other side, we're not doing the same for the banner URL. If the banner URL is not provided we just omit the property, so I see an inconsistency here

Reply:
> When it's not provided(on yt side) the backend returns a blank string

Correct, the BE returns the same response as it's returned by the YT api, so I made the property required since it will not be `undefined` , only a empty string or containing description.

So let me know if you want to omit the `channelDescription` if it was returned as empty/blank string by Youtube API

Reply:
hmm, no need. If this is the same response as it's returned by YT API,  let's leave it as it is